### PR TITLE
docs: Add issue template 

### PR DESCRIPTION
**Pull request by tanmay-pnaik**
## Overview

Add issue template

## Notes for reviewer

Adapted from https://github.com/bank-vaults/vault-operator/tree/main/.github/ISSUE_TEMPLATE

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
+name: 🐛 Bug report
+description: Report a bug to help us improve the Vault Helm Chart
+labels: [bug]
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:3]:
```suggestion
labels: [kind/bug]
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        If you are not sure if it is a bug or not, you can contact us via the available [support channels](https://github.com/bank-vaults/vault-helm-chart/issues/new/choose).
+  - type: checkboxes
+    attributes:
+      label: Preflight Checklist
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have searched the [issue tracker](https://www.github.com/bank-vaults/vault-helm-chart/issues) for an issue that matches the one I want to file, without success.
+          required: true
+        - label: I am not looking for support or already pursued the available [support channels](https://github.com/bank-vaults/vault-helm-chart/issues/new/choose) without success.
+          required: true
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:21]:
```suggestion
          required: true
        - label: I agree to follow the [Code of Conduct](https://bank-vaults.dev/docs/code-of-conduct/).
          required: true
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Installation Type
+      description: How did you install the Operator?
+      options:
+        - Official Helm chart
+        - Custom Helm chart
+        - Other (specify below)
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:43]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    attributes:
+      label: Configuration
+      description: Contents of Vault CustomResource.
+      render: yaml
+      placeholder: |
+        apiVersion: "vault.banzaicloud.com/v1alpha1"
+        kind: "Vault"
+        metadata:
+          name: "vault"
+        # ...
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:94]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+  - type: textarea
+    attributes:
+      label: Logs
+      description: Operator application logs (if relevant).
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:98]: Bank-Vaults? Vault?

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
+name: 🎉 Feature request
+description: Suggest an idea for the Vault Helm Chart
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:2]:
```suggestion
description: Suggest an idea for the Vault Helm Chart
labels: [kind/enhancement]
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+        Please describe what you would like to change/add and why in detail by filling out the template below.
+
+        If you are not sure if your request fits into Dex, you can contact us via the available [support channels](https://github.com/bank-vaults/vault-helm-chart/issues/new/choose).
+  - type: checkboxes
+    attributes:
+      label: Preflight Checklist
+      description: Please ensure you've completed all of the following.
+      options:
+        - label: I have searched the [issue tracker](https://www.github.com/bank-vaults/vault-helm-chart/issues) for an issue that matches the one I want to file, without success.
+          required: true
```
Comment (sagikazarmark) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:18]:
```suggestion
          required: true
        - label: I agree to follow the [Code of Conduct](https://bank-vaults.dev/docs/code-of-conduct/).
          required: true
```

**Review by sagikazarmark — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 5 above)
Comment (tanmay-pnaik) [reply .github/ISSUE_TEMPLATE/bug_report.yaml]: Changed to Vault. :+1:

**Review by ramizpolic — APPROVED** [review]

**Comment by ramizpolic** [comment]
@tanmay-pnaik could you do a rebase? I can merge it right away 😄

**Review by ramizpolic — CHANGES_REQUESTED** [review]
Git history seems to be corrupted, would you mind fixing/opening a new PR with selected changes only? Sorry for being a pain 😢

**Comment by tanmay-pnaik** [comment]
> Git history seems to be corrupted, would you mind fixing/opening a new PR with selected changes only? Sorry for being a pain 😢

No worries, sorry from my end too. Resolved now.

**Review by ramizpolic — APPROVED** [review]

**Pull request merged**